### PR TITLE
[Docs] Remove old "new" badges

### PR DESCRIPTION
**Pull request by heyjoethomas**
Remove new badges that were added with Uplift/Gen 3 documentation.

### Before
Screenshot 2024-09-19 at 11 07 27 AM
Screenshot 2024-09-19 at 11 07 31 AM

### After
Screenshot 2024-09-19 at 11 07 10 AM
Screenshot 2024-09-19 at 11 07 16 AM

Gen 3 has been out long enough the content is no longer "new" 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#install-dependencies-and-build-workspaces)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

### 🎩 checklist

- [ ] Tested a [snapshot](https://github.com/Shopify/polaris/blob/main/documentation/Releasing.md#-snapshot-releases)
- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

**Review by sarahill — APPROVED** [review]
✅

File: polaris.shopify.com/content/design/layout/index.mdx [hunk 1]
```diff
@@ -36,16 +35,18 @@ status: New
...
-    ![An order card with an inset table followed by an apps card with a grid
-    pattern made of different
-    surfaces](/images/design/layout/layout-overview-01-space@2x.png)
-  </Large>
+{' '}
+<Large>
+  ![An order card with an inset table followed by an apps card with a grid
+  pattern made of different
+  surfaces](/images/design/layout/layout-overview-01-space@2x.png)
+</Large>
```
Comment (sam-b-rose) [inline polaris.shopify.com/content/design/layout/index.mdx]:
```suggestion
  <Large>
    ![An order card with an inset table followed by an apps card with a grid
    pattern made of different
    surfaces](/images/design/layout/layout-overview-01-space@2x.png)
  </Large>
```

File: polaris.shopify.com/content/design/layout/index.mdx [hunk 2]
```diff
@@ -36,16 +35,18 @@ status: New
...
 
-  <Text>
-    Grouping similar items together helps merchants understand which elements are
-    related and helps them make faster decisions.
-  </Text>
+{' '}
+<Text>
+  Grouping similar items together helps merchants understand which elements are
+  related and helps them make faster decisions.
+</Text>
```
Comment (sam-b-rose) [inline polaris.shopify.com/content/design/layout/index.mdx]:
```suggestion
  <Text>
    Grouping similar items together helps merchants understand which elements are
    related and helps them make faster decisions.
  </Text>
```

File: polaris.shopify.com/content/design/layout/index.mdx [hunk 3]
```diff
@@ -108,16 +111,18 @@ status: New
...
+  ![Different cards showing button placement, popover contextual menus and
+  high-density cards containing a lot of
+  information](/images/design/layout/layout-overview-07-adapt@2x.png)
+</Large>
+
+{' '}
+<Text>
+  Compact elements add detail, and larger elements command more attention.
+  Surfaces adapt to these components and offer an optimized view of the admin.
+</Text>
```
Comment (sam-b-rose) [inline polaris.shopify.com/content/design/layout/index.mdx]:
```suggestion
  <Large>
    ![Different cards showing button placement, popover contextual menus and
    high-density cards containing a lot of
    information](/images/design/layout/layout-overview-07-adapt@2x.png)
  </Large>

  <Text>
    Compact elements add detail, and larger elements command more attention.
    Surfaces adapt to these components and offer an optimized view of the admin.
  </Text>
```

**Review by sam-b-rose — APPROVED** [review]
Looks good! Just added some suggestion comments to restore some formatting changes that were likely due to the automatic formatting

File: polaris.shopify.com/content/design/layout/index.mdx [hunk 4]
```diff
@@ -73,17 +72,19 @@ status: New
...
+  that have a title and a larger stat
+  numbe](/images/design/layout/layout-overview-04-emphasis@2x.png)
+</Large>
+
+{' '}
+<Text>
+  Use size, weight, and contrast to establish hierarchy in the admin. Divider
+  lines are used to delimit rows of information in data and index tables, and
+  rarely for dividing information elsewhere.
+</Text>
```
Comment (sam-b-rose) [inline polaris.shopify.com/content/design/layout/index.mdx]:
```suggestion
  <Large>
    ![A grid pattern showing different marketing statistics, separated in cards
    that have a title and a larger stat
    numbe](/images/design/layout/layout-overview-04-emphasis@2x.png)
  </Large>

  <Text>
    Use size, weight, and contrast to establish hierarchy in the admin. Divider
    lines are used to delimit rows of information in data and index tables, and
    rarely for dividing information elsewhere.
  </Text>
```

**Pull request merged**